src/graphics/ui/eventListeners.ts

```diff
@@ -851,6 +850,9 @@ export function clickHandler(overworld: Overworld, e: MouseEvent) {
       playSFXKey('deny');
     }
   } else {
+
+    const mouseTarget = underworld.getMousePos();
+    drawCircleUnderTarget(mouseTarget, underworld, 1.0, globalThis.planningViewGraphics);
```

Comment:
What is this for? Click handler just triggers once on click and then is done.  Persistent circles on screen are usually drawn inside of recurring functions such as gameLoop or something that is called every frame

Reply:
Um... I'm not really sure. In some situations where units and pickups were nearby, there was a discrepancy between what highlighted when you hovered, and what you actually selected when you clicked. I think I was testing something here and forgot to remove it.